### PR TITLE
Fixes Eminence Bugs

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Fixes Servant Warp and Manipulate Reality.

## Why It's Good For The Game
Spells that don't work aren't good spells.

## Changelog
:cl:
fix: Fixed the Eminence's Manipulate Reality and Servant Warp spells
/:cl:

### CODE REVIEW

File: code/modules/antagonists/clock_cult/mobs/eminence.dm [hunk 1]
```diff
@@ -293,8 +290,6 @@
 		"Processor Overload",
 		"Radiation Storm"
 	)
-	if(!can_cast(user))
-		return
```
Comment (KubeRoot) [inline code/modules/antagonists/clock_cult/mobs/eminence.dm:297]: Removing this might be a bad idea, maybe `cast_check(TRUE, user)` could work here?

Comment (Ivniinvi) [reply code/modules/antagonists/clock_cult/mobs/eminence.dm:297]: I'm pretty sure that can_cast is run elsewhere earlier in the call chain of the button.

Comment (PowerfulBacon) [reply code/modules/antagonists/clock_cult/mobs/eminence.dm:297]: Yea but it needs to be here because it is after a user input which is blocking

**Review by PowerfulBacon — DISMISSED** [review]
Actually it should be fine since it passed the check anyway to get to that point

File: code/modules/antagonists/clock_cult/mobs/eminence.dm (hunk 1 above)
Comment (ike709) [reply code/modules/antagonists/clock_cult/mobs/eminence.dm:297]: Hello?

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**